src/Files.App/Data/Commands/CommandCodes.cs

```diff
@@ -41,6 +41,7 @@ public enum CommandCodes
 		CreateShortcutFromDialog,
 		EmptyRecycleBin,
 		FormatDrive,
+		OpenStorageSense,
```

Comment:
I would put this under "Open"

Reply:
This is somewhat related to FileSystem. But ok.

Reply:
Done